src/Libraries/GeometryUI/PanelSurfaceBoundaryConditions.cs

```diff
@@ -0,0 +1,79 @@
+using Autodesk.DesignScript.Geometry;
+using Autodesk.DesignScript.Runtime;
+using CoreNodeModels;
+using DSCore;
+using Dynamo.Graph.Nodes;
+using Dynamo.Utilities;
+using Newtonsoft.Json;
+using ProtoCore.AST.AssociativeAST;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GeometryUI
+{
+    [IsVisibleInDynamoLibrary(true)]
+    [NodeName("PanelSurfaceBoundaryCondition")]
+    [NodeCategory("Geometry.PanelSurface")]
+    [NodeDescription("PanelSurfaceBoundaryConditionDropDownDesc", typeof(Properties.Resources))]
+    [OutPortNames(">")]
```

Comment:
Can the outport name also be `panelsurfaceboundarycondition`?

Reply:
If you think it makes it clearer. 
I just found the node gets a lot larger because of the extra characters

Reply:
I guess you're right. Let's leave it as-is.